Indicators/cmf.py

```diff
@@ -0,0 +1,32 @@
+import pandas as pd
+
+class CMF:
+    data_name = 'cs-cmf.xls'
```

Comment:
```suggestion
    data_name = 'cs-cmf.xls'
    data = pd.read_excel(data_name)
    # Column adlarını düzeltme
    data.columns.values[4] = "MF_Multiplier"
    data.columns.values[6] = "MF_Volume"
    data.columns.values[7] = "20_period_CMF"
```
bize gelen veri excel dosyasından değilde dışarıdan geleceği için 4-9 arası gerek yok silebilirsin